frontend/Draggable.js

```diff
@@ -15,8 +15,8 @@ var ReactDOM = require('react-dom');
 import type {DOMEvent} from './types';
 
 class Draggable extends React.Component {
-  _onMove: (evt: DOMEvent) => void;
-  _onUp: (evt: DOMEvent) => void;
+  _onMove: (evt: any) => void;
+  _onUp: (evt: any) => void;
```

Comment:
Those caused me the most problems. I'm not quite sure how to solve it properly, because if you substitute `any` with `DOMEvent` you'll get a huge error message about not being able to call `addEventListener`/`removeEventListener` on all elements of the intersection